src/validation.h

```diff
@@ -109,6 +109,7 @@ extern RecursiveMutex cs_main;
 typedef std::unordered_map<uint256, CBlockIndex*, BlockHasher> BlockMap;
 extern Mutex g_best_block_mutex;
 extern std::condition_variable g_best_block_cv;
+/** Used to notify getblocktemplate RPC of new tips. */
```

Comment:
nit: why this comment here?

Reply:
I noticed it was undocumented and this is set by `UpdateTip` so it's tangentially related... but agree it looks confusing in this commit. I'll break it out into a separate commit.